canonical/lxd-ui
# feat: improve permission selector [WD-15106]

## Done

- Improve permission resource selector
- Improve permission entitlement selector

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @mas-who or @edlerd for access.
    - With a local copy of this branch, [build and run as described in the docs](../CONTRIBUTING.md#setting-up-for-development).
2. Perform the following QA steps:
    - play around with the permission selector and make sure it works the same (functionally)

## Screenshots
![Screenshot from 2024-10-03 21-02-02](https://github.com/user-attachments/assets/5f04dcb4-4815-4e1c-b1ae-94e4c440025b)
![Screenshot from 2024-10-04 18-38-37](https://github.com/user-attachments/assets/680be476-14ce-45e8-97cc-23339d97efab)
